src/devices/DCMotor/DCMotor.cs

```diff
@@ -194,7 +208,7 @@ public static DCMotor Create(int speedControlPin, int directionPin, GpioControll
         /// <paramref name="otherDirectionPin"/> should be connected to H-bridge input corresponding to the remaining motor input.
         /// Connecting motor directly to GPIO pin is not recommended and may damage your board.
         /// </remarks>
-        public static DCMotor Create(PwmChannel speedControlChannel, int directionPin, int otherDirectionPin, GpioController controller = null, bool shouldDispose = true)
+        public static DCMotor Create3pinDriver(PwmChannel speedControlChannel, int directionPin, int otherDirectionPin, GpioController controller = null, bool shouldDispose = true)
```

Comment:
whatever name you pick for the new driver, this name should also change as it's similar concept except it's using 2 pins for direction rather than one